_rules/document-has-instrument-to-non-repeated-content-ye5d6e.md

```diff
@@ -0,0 +1,415 @@
+---
+id: ye5d6e
+name: Document has an instrument to move focus to non-repeated content
+rule_type: atomic
+description: |
+  This rule checks that there is an instrument to move focus to non-repeated content in the page
+accessibility_requirements:
+  wcag-technique:G1: # Adding a link at the top of each page that goes directly to the main content area
+    forConformance: false
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+  wcag-technique:G123: # Adding a link at the beginning of a block of repeated content to go to the end of the block
+    forConformance: false
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+  wcag-technique:G124: # Adding links at the top of the page to each area of the content
+    forConformance: false
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - Accessibility Tree
+  - CSS styling
+  - DOM Tree
+  - Language
+acknowledgments:
+  authors:
+    - Jean-Yves Moyen
+  assets:
+    - _The Three Kingdoms_ by Luo Guanzhong, translation by Yu Sumei (Tuttle publishing, 2014, ISBN 9780804843935)
+---
+
+## Applicability
+
+This rule applies to any [HTML web page][].
+
+## Expectation
+
+For each test target, there exists at least one [instrument][] inside it to move focus [just before][] a node of [non-repeated content after repeated content][].
```

Comment:
This bit...

> [...] exists at least one instrument inside it [...]

...seems like it needs to be better qualified to ensure unambiguity.

Reply:
This is actually part of [Web page (HTML)](https://act-rules.github.io/glossary/#web-page-html) (third note).
Agreed, it is written in an old style and could use some upgrade, but "element in a web page" is unambiguous and objective 😀

Reply:
Yeah, that's not at all obvious 🙈 Oh well 🤷